README.md

```diff
@@ -7,15 +7,15 @@ For more information about Keycloak visit the [Keycloak homepage](http://keycloa
 
 
 Contributing to Keycloak Documentation
---------------------------------------
+----------------------------------------
 
 See our [Contributor's Guide](internal_resources/contributing.adoc). The directory also includes a set of templates and other resources to help you get started.
 
 If you want to file a bug report or tell us about any other issue with this documentation, you are invited to please use our [issue tracker](https://issues.redhat.com/projects/KEYCLOAK/).
 
 
 Building Keycloak Documentation
--------------------------------
+---------------------------------
```

Comment:
Just a nitpick (not a blocker). Why to make this change? Seems unrelated (and even in an unrelated file).